packages/actor_pinning/auth.pony

```diff
@@ -0,0 +1,3 @@
+primitive PinUnpinActorAuth
```

Comment:
I think calling this PinActorAuth is fine and reads easily enough. Whereas PinUnpinActorAuth is a little hard for me to parse. But I'm fine either way. 

I'd like to hear what other @ponylang/core folks have to say.

Reply:
that's a good precedent.

Reply:
i agree.. the guy who did that is smart and should be listened to..